src/application/modules/legacy/types.ts

```diff
@@ -16,3 +16,19 @@ export interface LegacyStoreData {
 	legacyAddress: string;
 	balance: bigint;
 }
+
+export interface ReclaimParamData {
```

Comment:
Let us keep the naming consistent throughout.

```suggestion
export interface ReclaimParamsData {
```